problems building per README

The README refers to using the `goapp` program but I do not seem to have it. I installed the Google Cloud SDK on MacOS today, and also installed the app-engine-go components. What is the workaround?   `go get -d` runs cleanly.  But `go build` from the app directory gives me this:

```
$ go build
../../../mjibson/goread/main.go:35:2: cannot find package "appengine" in any of:
	/Users/fred/sdk/go1.11.9/src/appengine (from $GOROOT)
	/Users/fred/go/src/appengine (from $GOPATH)
 ...
```

(My apologies if this is a rookie kind of problem. I have not built for appengine in many years.)

--- a/README.md
+++ b/README.md
@@ -12,8 +12,6 @@ by [Michael Blakeley](https://github.com/mblakele).
 1. Install [Git](http://gitscm.com/) and [Mercurial](http://mercurial.selenic.com/wiki/Download) and make sure `git` and `hg` are in your `PATH`.
 1. Install the [Go App Engine SDK](https://developers.google.com/appengine/downloads#Google_App_Engine_SDK_for_Go).
 1. Set your `GOPATH` (to something like `/home/user`), and make sure it's a directory that exists. (Note: set this on your machine's environment, not in the go.bat file.)
-1. Further commands use `goapp` which lives in the `go_appengine` directory from the SDK. Make sure it's in your `PATH`.
-1. Download goread and dependencies by running: `goapp get -d github.com/mblakele/goread`. You may get messages about unrecognized imports. Ignore them.
 1. `cd $GOPATH/src/github.com/mblakele/goread/app`.
 1. Copy `app.sample.yaml` to `app.yaml`.
 1. `cd ..`

--- a/app/app.go
+++ b/app/app.go
@@ -5,7 +5,7 @@ import (
 
 	"github.com/gorilla/mux"
 
-	app "github.com/mjibson/goread"
+	app "github.com/mblakele/goread"
 
 	"google.golang.org/appengine"
 )

--- a/app/templates/base.html
+++ b/app/templates/base.html
@@ -362,7 +362,7 @@
 			<div class="col-md-offset-3 col-md-6">
 				<h1>Account</h1>
 				<div ng-hide="accountType">
-					<p>Go Read's <a href="https://github.com/mjibson/goread">software</a> is free, but its servers are not. A 30-day free trial is provided, after which we require a paid subscription.</p>
+					<p>Go Read's <a href="https://github.com/mblakele/goread">software</a> is free, but you may have to pay Google to run it.</p>
 					<div>
 						{{range .StripePlans}}
 						<button
@@ -403,13 +403,11 @@
 					Built by <a href="http://twitter.com/mjibson">Matt Jibson</a>.
 					Runs on <a href="https://developers.google.com/appengine/">Google App Engine</a>.
 					Powered by <a href="http://golang.org/">Go</a> and <a href="http://angularjs.org/">AngularJS</a>.
-					Code on <a href="https://github.com/mjibson/goread">GitHub</a>.
+					Code on <a href="https://github.com/mblakele/goread">GitHub</a>.
 				</p>
 				<h1>Help</h1>
-				<p>Problems? Try one of these:</p>
-				<p>Tweet <a href="https://twitter.com/goreadrss">@GoReadRss</a></p>
-				<p>Email <a href="mailto:support@goread.io">support@goread.io</a></p>
-				<p>Submit a <a href="https://github.com/mjibson/goread/issues">GitHub issue</a></p>
+				<p>Problems?</p>
+				<p>Submit a <a href="https://github.com/mblakele/goread/issues">GitHub issue</a></p>
 				<h1>FAQ</h1>
 				<p>
 					<strong>Are there keyboard shortcuts?</strong>
@@ -417,7 +415,7 @@
 				</p>
 				<p>
 					<strong>What is on your todo list?</strong>
-					Check the <a href="https://github.com/mjibson/goread/issues">GitHub issues</a> page.
+					Check the <a href="https://github.com/mblakele/goread/issues">GitHub issues</a> page.
 				</p>
 				<p>
 					<strong>Is there a bookmarklet/submit URL?</strong>
@@ -430,7 +428,7 @@
 				<p>
 					<strong>How much does a paid subscription cost?</strong>
 					Either $3/month or $30/year. A 30-day free trial is provided. Please contact support if you have questions.
-					If you are not able to pay, setting up your own server is not difficult (and free for a few hundred feeds). Follow the instructions on the <a href="https://github.com/mjibson/goread">GitHub page</a>. 
+					If you are not able to pay, setting up your own server is not difficult (and free for a few hundred feeds). Follow the instructions on the <a href="https://github.com/mblakele/goread">GitHub page</a>. 
 				</p>
 				{{end}}
 			</div>

--- a/user.go
+++ b/user.go
@@ -34,8 +34,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/mblakele/goread/sanitizer"
 	"github.com/mjibson/goon"
-	"github.com/mjibson/goread/sanitizer"
 	"golang.org/x/net/html/charset"
 
 	"google.golang.org/appengine"

--- a/utils.go
+++ b/utils.go
@@ -31,11 +31,13 @@ import (
 	"time"
 	"unicode/utf8"
 
+	"github.com/mblakele/goread/atom"
+	"github.com/mblakele/goread/rdf"
+	"github.com/mblakele/goread/rss"
+	"github.com/mblakele/goread/sanitizer"
+
 	"github.com/mjibson/goon"
-	"github.com/mjibson/goread/atom"
-	"github.com/mjibson/goread/rdf"
-	"github.com/mjibson/goread/rss"
-	"github.com/mjibson/goread/sanitizer"
+
 	"golang.org/x/net/html/charset"
 	"golang.org/x/text/encoding"
 	"golang.org/x/text/encoding/charmap"

